cms/check_requests_count_volume

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# Copyright 2012-2024 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Christos Emmanouil, <christos.emmanouil@cern.ch>, 2024
+
+"""
+Probe to check number and volume of requests per rses, account, activity and state.
+"""
+
+import sys
+import traceback
+
+from sqlalchemy import func
+from sqlalchemy.orm import aliased
+
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+from rucio.common.types import InternalScope
+
+from utils import common
+PrometheusPusher = common.PrometheusPusher
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if __name__ == "__main__":
+
+    try:
+        
+        session = get_session()
+        
+        DestRSE = aliased(models.RSE)
+        SourceRSE = aliased(models.RSE)
+
+        requests_query = (
+            session.query(
+                SourceRSE.rse.label("source_rse"),
+                DestRSE.rse.label("dest_rse"),
+                models.Request.account,
+                models.Request.activity,
+                models.Request.state,
+                models.Request.external_host,
+                func.count(func.distinct(models.Request.id)),
+                func.sum(models.Request.bytes)
+            )
+            .with_hint(DestRSE, "INDEX_FFS(rses RSES_PK)", "oracle")
+            .with_hint(SourceRSE, "INDEX_FFS(rses RSES_PK)", "oracle")
+            .with_hint(models.Request, 'INDEX_FFS(requests REQUESTS_PK)', 'oracle')
+            .with_hint(models.Request, 'INDEX_FFS(requests REQUESTS_RULEID_IDX)', 'oracle')
+            .with_hint(models.Request, 'INDEX_FFS(requests REQUESTS_DEST_RSE_ID_IDX)', 'oracle')
+            .with_hint(models.Request, 'INDEX_FFS(requests REQUESTS_TYP_STA_UPD_IDX)', 'oracle')
+            .select_from(models.Request)
+            .filter(models.Request.scope == InternalScope("cms"))
+            .join(DestRSE, models.Request.dest_rse_id == DestRSE.id)
+            .join(SourceRSE, models.Request.source_rse_id == SourceRSE.id)
+            .group_by(
+                SourceRSE.rse,
+                DestRSE.rse,
+                models.Request.account,
+                models.Request.activity,
+                models.Request.state,
+                models.Request.external_host
+            )
+        )
+        
+        with PrometheusPusher() as manager:
+            for (
+                source_rse, dest_rse, account, activity, 
```

Comment:
Stylistically don't put this on a new line, continue from above and then ident source_rse and requests_state to the same level. You can also combine lines 82 and 83.